scripts/support/create-support-bundle.sh

```diff
@@ -0,0 +1,313 @@
+    fi
+  fi
+}
+
+os_check() {
+  info "Checking that the operating system is supported..."
+  os_type=$(uname -s)
+  case "$os_type" in
+    Linux)
+      succeeded
+      ;;
+    *)
+      failed
+      error_exit "$LINENO" "The operating system $(fg_yellow "$os_type") is not supported by this script."
+      ;;
+  esac
+}
+
+os_arch_check() {
+  info "Checking for valid operating system architecture..."
+  arch=$(uname -m)
+  case "$arch" in
+    x86_64|amd64)
+      arch=amd64
+      ;;
+    aarch64|arm64|aarch64_be|armv8b|armv8l)
+      arch=arm64
+      succeeded
+      ;;
+    *)
+      failed
+      error_exit "$LINENO" "The operating system architecture $(fg_yellow "$arch") is not supported by this script."
+      ;;
+  esac
+}
+
+
+# This will check if the current environment has
+# all required shell dependencies to run the installation.
+dependencies_check() {
+  info "Checking for script dependencies..."
+  FAILED_PREREQS=''
+  for prerequisite in $PREREQS; do
+    if command -v "$prerequisite" >/dev/null; then
+      continue
+    else
+      if [ -z "$FAILED_PREREQS" ]; then
+        FAILED_PREREQS="${fg_red}$prerequisite${reset}"
+      else
+        FAILED_PREREQS="$FAILED_PREREQS, ${fg_red}$prerequisite${reset}"
+      fi
+    fi
+  done
+
+  if [ -n "$FAILED_PREREQS" ]; then
+    failed
+    error_exit "$LINENO" "The following dependencies are required by this script: [$FAILED_PREREQS]"
+  fi
+  succeeded
+}
+
+check_prereqs() {
+  banner "Checking Prerequisites"
+  increase_indent
+  root_check
+  os_check
+  os_arch_check
+  dependencies_check
+  success "Prerequisite check complete!"
+  decrease_indent
+}
+
+function bundle_files() {
+    banner "Collecting files for support bundle"
+    increase_indent
+    # Directory for logs
+    log_dir="/opt/observiq-otel-collector/log"
+    
+    # Check if directory exists
+    if [ ! -d "$log_dir" ]; then
+        info "Directory ($fg_red "$log_dir")$(reset) does not exist."
+        read -p "Please enter an existing directory for logs: " log_dir
+        if [ ! -d "$log_dir" ]; then
+            echo "Directory $log_dir does not exist."
+            return 1
+        fi
+    fi
+
+    read -p "Do you want to include only the most recent logs (y or n)? " response
+    increase_indent
+    tar_filename="support_bundle_$(date +%Y%m%d_%H%M%S).tar"
+    if [ "$response" = "n" ]; then
+        # Get all the log files
+        info "Collecting all log files in $(fg_cyan "$log_dir")$(reset)"
+        sudo tar -cvf $tar_filename $log_dir
+    else
+        # Get the most recent log file
+        recent_log=$(ls -Art $log_dir | tail -n 1)
+        if [ -n "$recent_log" ]; then
+            sudo tar -cvf $tar_filename "$log_dir/$recent_log" 
+            info "Added file $(fg_cyan "$recent_log")$(reset) to the tar file."
+
+        else
+            info "No logs found in $(fg_red $log_dir)"
+            return 1
+        fi
+    fi
```

Comment:
Not sure where this should be, but I think we should also capture some output from journalctl (`journalctl -u observiq-otel-collector.service -n 500` or similar), these logs may have useful panic information.